cmd/api/src/queries/graph_test.go

```diff
@@ -141,28 +142,44 @@ func TestGraphQuery_RawCypherSearch(t *testing.T) {
 			txConfig := &graph.TransactionConfig{}
 			options[0](txConfig)
 
-			require.Equal(t, time.Minute*5, txConfig.Timeout)
+			require.Equal(t, time.Second*225, txConfig.Timeout)
```

Comment:
I'm not sure I understand why this went from 5m to 3m45s, do we know what's causing this change?

Reply:
Ah, that makes sense. It was hard to correlate just by looking at the test, but now I get it. A proper set of tests would be nice for sure